README.md

```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
 * [Intro to Zero-Knowledge Proofs](https://www.quicknode.com/guides/knowledge-base/intro-to-zero-knowledge-proofs) - short intro to the subject by Ferhat Kochan. 
 * [The Complete Full-Stack Guide to Getting Started with Zero-Knowledge Proofs using Circom and ZK-Snarks](https://dev.to/tonyolendo/the-complete-full-stack-guide-to-getting-started-with-zero-knowledge-proofs-using-circom-and-zk-snarks-part-1-53gi) - Full-Stack Guide for beginners by Tony Olendo.
 * [A Simple Explanation of Zero Knowledge Proofs](https://medium.com/web3studio/a-simple-explanation-of-zero-knowledge-proofs-ca574092e73b) - Short explanation of the ZKP topic by Hadas Zeilberger.
-* [Understanding Zero-Knowledge Proofs Through the Source Code of Tornado Cash](https://betterprogramming.pub/understanding-zero-knowledge-proofs-through-the-source-code-of-tornado-cash-41d335c5475f) - explanation of the ZKP with example on Tornado Cash by Laszlo Fazekas.
-* [The Evolution of Zero-Knowledge Proofs in Web3](https://medium.com/manta-network/the-evolution-of-zero-knowledge-proofs-in-web3-b92a68c41c04) - description of the ZKP with example on Manta Network.
-* [ZKP Knowledge Base](https://kb.delendum.xyz/) - knowledge base wikia by Delendum 
+* [Understanding Zero-Knowledge Proofs Through the Source Code of Tornado Cash](https://betterprogramming.pub/understanding-zero-knowledge-proofs-through-the-source-code-of-tornado-cash-41d335c5475f) - explanation of the ZKP with an example on Tornado Cash by Laszlo Fazekas.
+* [The Evolution of Zero-Knowledge Proofs in Web3](https://medium.com/manta-network/the-evolution-of-zero-knowledge-proofs-in-web3-b92a68c41c04) - description of the ZKP with an example on Manta Network.
+* [ZKP Knowledge Base](https://kb.delendum.xyz/) - knowledge base Wikia by Delendum 
 * [Commitment Schemes and Zero-Knowledge Protocols, January 1998](https://www.researchgate.net/publication/221621428_Commitment_Schemes_and_Zero-Knowledge_Protocols) - academic text by Ivan Damgård from Aarhus University. 
 * [The Complexity of Zero Knowledge by Harvard University, October 2007](https://people.seas.harvard.edu/~salil/research/complexityZK.pdf) - academic text by Salil Vadhan from Harvard University. 
 
-### Tutorials and courses
-* [Zero Knowledge Proofs](https://zk-learning.org/) - course on Spring 2023, made by MOOC
+### Tutorials
 * [Zero Knowledge on Casper](https://odra.dev/blog/casper-zk-risc0) - implementing ZK on Casper using risc0.
+* [ZKSnark tutorial](https://github.com/TheBojda/zksnark-tutorial) - JavaScript tutorial for Zero-Knowledge Proofs using SnarkJS and Circom.
```

Comment:
Is there additional documentation for this? There is only some code without explanation. Not a tutorial imo.

Reply:
That's a good point. Fixed.